compiler-plugin/testData/api/implicitElements.kt

```diff
@@ -22,10 +22,10 @@
 // <null>
 // <null>
 // <init>,<init>,<init>
-// readOnly.get(): SYNTHETIC annotations from property: GetAnno
-// readOnly.getter.owner: readOnly: KOTLIN
+// readOnly.get(): CLASS annotations from property: GetAnno
+// readOnly.getter.owner: readOnly: CLASS
```

Comment:
This does not look right, as the owner should be `KOTLIN`

Reply:
i think i can fix this by still using the synthetic but constructing it w/ the descriptor.
It might be a bit weird because we already have the descriptor when i create it but synthetic only expects the ktProperty, i can try to see how i can avoid the re-query there.